package.json

```diff
@@ -8,20 +8,7 @@
     "build": "ng build --app=foundation-demo",
     "test": "ng test",
     "lint": "./node_modules/.bin/nx lint && ng lint",
```

Comment:
"./node_modules/.bin/nx" can be removed, right?

Reply:
@lderrickable I updated the PR.